planner/core/common_plans.go

```diff
@@ -1275,8 +1279,12 @@ func (e *Explain) prepareOperatorInfo(p Plan, taskType, driverSide, indent strin
 		actRows, analyzeInfo, memoryInfo, diskInfo := getRuntimeInfo(e.ctx, p, nil)
 		row = []string{id, estRows, actRows, taskType, accessObject, analyzeInfo, operatorInfo, memoryInfo, diskInfo}
 	} else if e.RuntimeStatsColl != nil {
+		row = []string{id, estRows}
```

Comment:
Do we also need to handle the case `e.Analyze`?

Reply:
Now `explain analyze` + `format = 'verbose'` seems not to be support yet. So I am not implementation this. What's your opinion? @xuyifangreeneyes

Reply:
I am ok with that. Can you leave a comment so that it may be easy for others to know the situation?